roles/ceph-rgw/tasks/multisite/secondary.yml

```diff
@@ -10,7 +17,10 @@
 - name: ensure connection to primary cluster from rgw
   uri:
     url: "{{ item.endpoint }}"
-  loop: "{{ rgw_instances }}"
+  loop: "{{ secondary_realms }}"
+  when:
+    - secondary_realms is defined
+    - item.host == inventory_hostname
```

Comment:
the main issue to iterate over the `secondary_realms` is that each rgw node will test ALL endpoints and not only the one used by the local instances.
So I think we should rollback this change and use `rgw_instances` instead.